pkgs/development/python-modules/py-libzfs/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, cython
+, zfs
+}:
+
+buildPythonPackage rec {
+  pname = "py-libzfs";
+  version = "22.2.1";
```

Comment:
The latest release seems to be [22.2.4](https://github.com/truenas/py-libzfs/releases/tag/TS-22.02.4).

Reply:
Oh yes, not on the master branch so I didn't noticed that. Seems like they're using gitflow workflow.